controller/monitor/node_monitor.go

```diff
@@ -98,9 +98,13 @@ func (m *NodeMonitor) GetCollectedData() (interface{}, error) {
 	m.collectedDataLock.RLock()
 	defer m.collectedDataLock.RUnlock()
 
-	data := make(map[string]*CollectedDiskInfo, 0)
-	if err := copier.Copy(&data, &m.collectedData); err != nil {
-		return data, errors.Wrap(err, "failed to copy node monitor collected data")
+	v, err := copystructure.Copy(m.collectedData)
```

Comment:
It's OK, but FYI. the recent version of copier has deepcopy as well, https://github.com/jinzhu/copier/blob/master/copier.go#L74.

Reply:
Let me update to latest copier

Reply:
Addressed!